hw/formal/tools/jaspergold/parse-fpv-report.py

```diff
@@ -102,15 +102,19 @@ def get_results(logpath):
     return results
 
 
-def get_cov_results(logpath):
+def get_cov_results(logpath, dut_name):
     '''Parse FPV coverage information from the log file'''
     try:
         with Path(logpath).open() as f:
             full_file = f.read()
-            coverage_patterns = [("stimuli", r"^\|\w+.*\|(\d*.\d\d)%.*\|\d*.\d\d%.*\|\d*.\d\d%.*"),
-                                 ("coi", r"^\|\w+.*\|\d*.\d\d%.*\|(\d*.\d\d)%.*\|\d*.\d\d%.*"),
-                                 ("proof", r"^\|\w+.*\|\d*.\d\d%.*\|\d*.\d\d%.*\|(\d*.\d\d)%.*")]
+            cov_pattern = r"\s*\|\d*.\d\d%\s\(\d*\/\d*\)"  # cov pattern: 100.00% (5/5)
+            pattern_match = r"\s*\|(\d*.\d\d)%\s\(\d*\/\d*\)"  # extract percentage in cov_pattern
+            coverage_patterns = \
+                [("stimuli", r"^\|" + dut_name + pattern_match + cov_pattern + cov_pattern),
+                 ("coi", r"^\|" + dut_name + cov_pattern + pattern_match + cov_pattern),
+                 ("proof", r"^\|" + dut_name + cov_pattern + cov_pattern + pattern_match)]
             cov_results = extract_messages(full_file, coverage_patterns)
+            print(cov_results)
```

Comment:
nit: is this just for debug? or use log.info instead

Reply:
Thanks, this is for debug. I will remove this.